### DISCUSSION TITLE
WatchDogTimer on PoE board

### DISCUSSION THREAD

**Discussion by F4HUY** [lora-aprs/LoRa_APRS_iGate#141] [Q&A] [answered]
Hi all,

I have a TTGO PoE board installed on top of mast, I have some case of freezing (rarely but can happen) , so need unplug replug power supply,
I think its easy to put a WDT on ESP32, but not found information for platformIO, only Arduino
https://iotassistant.io/esp32/enable-hardware-watchdog-timer-esp32-arduino-ide/

any help/idea ?

thx you

**Comment by peterus** [comment, marked as answer]
there is a Ticket for WDT: https://github.com/lora-aprs/LoRa_APRS_iGate/issues/95
